v0/destinations/singular/util.js

```diff
@@ -138,6 +139,8 @@ const platformWisePayloadGenerator = (message, isSessionEvent) => {
     } else {
       payload.dnt = 1;
     }
+    // by default, the value of openuri should be "", i.e empty string if nothing is passed
+    payload.openuri = message.properties.url || "";
```

Comment:
This is same as
```
{
    "destKey": "openuri",
    "sourceKeys": "properties.url",
    "required": true,
    "metadata": {
      "defaultValue": ""
    }
  }
  ```
  Shall we use mapping json way only?

Reply:
Will "defaultValue" work with "required" mappings? I didn't see it working, that's why used the other way.

Reply:
We can remove "required" then

Reply:
It is not working.

Reply:
ok then its fine with current implementation